app/javascript/App/bundles/CoinShow/HistoricalPriceDataTable.tsx

```diff
@@ -101,36 +99,36 @@ class HistoricalPriceDataTable extends React.Component<Props, State> {
     // Set initial status
     const statusIsReady = !_.isUndefined(initialSortedData)
     const initialStatus = statusIsReady ? STATUSES.READY : STATUSES.INITIALIZING
-    const currency = initialData
-      ? _.get(initialData, ['0', 'to_currency'])
-      : undefined
 
     this.state = {
       status: initialStatus,
       data: initialSortedData || [],
-      currency,
+      rawData: initialData || [],
```

Comment:
Is it better to call this `initialRawData` to make the distinction between this `rawData` field and `data`

Reply:
Since it's state, I can see the rawData being updated asynchronously at some point and won't necessary only be initialRawData.

Reply:
Put the comment on the wrong spot really, I meant renaming the `prop.initialData` to `prop.initialRawData`